releases/nightly-build/compose-files/docker-compose-nexus-mongo-arm64.yml

```diff
@@ -292,17 +292,16 @@ services:
       - edgex-network
     environment:
       <<: *common-variables
-      Service_Host: edgex-support-logging
-      Writable_Persistence: database
       SecretStore_TokenFile: /tmp/edgex/secrets/edgex-support-logging/secrets-token.json
+      Service_Host: edgex-support-logging
+      Writable_Persistence: file
+      Databases_Primary_Type: file
       Logging_EnableRemote: "false"
-      Logging_File: /edgex/logs/edgex-support-logging.log
+      Logging_File: ""
```

Comment:
@andresrinivasan , @bnevis-i , Sorry that I am late to seeing this. Docker compose ignores empty environment variables like this.  Just remove the override since File is set to empty in the config file, so doesn't need to be overridden. Here is to log output with this changed to Logging_File is missing from overrides applied.

```
"Loaded configuration from /res/configuration.toml"
level=INFO ts=2020-04-24T15:27:58.808839799Z app=edgex-support-logging source=environment.go:327 msg="Environment override of 'Registry.Host' by environment variable: Registry_Host=edgex-core-consul"
level=INFO ts=2020-04-24T15:27:58.809151599Z app=edgex-support-logging source=environment.go:327 msg="Environment override of 'Databases.Primary.Port' by environment variable: Databases_Primary_Port=27017"
level=INFO ts=2020-04-24T15:27:58.809379699Z app=edgex-support-logging source=environment.go:327 msg="Environment override of 'Logging.EnableRemote' by environment variable: Logging_EnableRemote=false"
level=INFO ts=2020-04-24T15:27:58.809600599Z app=edgex-support-logging source=environment.go:327 msg="Environment override of 'Databases.Primary.Type' by environment variable: Databases_Primary_Type=file"
level=INFO ts=2020-04-24T15:27:58.809819799Z app=edgex-support-logging source=environment.go:327 msg="Environment override of 'Databases.Primary.Host' by environment variable: Databases_Primary_Host=edgex-mongo"
level=INFO ts=2020-04-24T15:27:58.810044199Z app=edgex-support-logging source=environment.go:327 msg="Environment override of 'Service.Host' by environment variable: Service_Host=edgex-support-logging"
level=INFO ts=2020-04-24T15:27:58.810215799Z app=edgex-support-logging source=environment.go:327 msg="Environment override of 'Writable.Persistence' by environment variable: Writable_Persistence=file"
```

Reply:
I'll let Andre make the changes and regress it.  In any case, despite the above error all the blackbox tests ran yesterday with >95% pass rate so we're headed in the right direction.

Reply:
I'll get to this in the late afternoon PT.

Reply:
See [PR #264](https://github.com/edgexfoundry/developer-scripts/pull/264).